src/test/java/de/uhd/ifi/se/decision/management/jira/view/TestChartCreator.java

```diff
@@ -0,0 +1,45 @@
+package de.uhd.ifi.se.decision.management.jira.view;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.atlassian.jira.project.MockProject;
+
+public class TestChartCreator {
+
+	ChartCreator creator;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/UHD/cures-condec-jira/pullRequest?prid=4666670)